tests/conftest.py

```diff
@@ -168,6 +171,11 @@ def catalog(id_generator, metastore, warehouse):
     return Catalog(id_generator=id_generator, metastore=metastore, warehouse=warehouse)
 
 
+@pytest.fixture
+def test_session(catalog):
```

Comment:
[Q] How does this fit with the `autouse=True` - `clean_session` fixture? Do you know if we still need that fixture now?

Reply:
Would using the following code mean that we could simplify these fixtures a little?

```python
with Session("TestSession", catalog=catalog) as session: 
    yield session
```

Reply:
[Q] and (this is just a suggestion as it might be a bit too "magic") but could we change `Session.get` to return the `test_session` (e.g. use environment variable or make this session the global one) and remove the need to always declare `test_session` and pass it?